storage/pkg/config/config.go

```diff
@@ -46,14 +46,19 @@ type Port struct {
 	// MaxCPUs can be a number or a percentage
 	MaxCPUs  string
 	LogLevel string
-	// Network can be tcp, udp or unix
-	Network string
-	// Addr to listen on, hostname:port (0.0.0.0:9999 for all interfaces) or socket (/var/run/reva/sock)
-	Addr string
+	// GRPCNetwork can be tcp, udp or unix
+	GRPCNetwork string
+	// GRPCAddr to listen on, hostname:port (0.0.0.0:9999 for all interfaces) or socket (/var/run/reva/sock)
+	GRPCAddr string
+	// Protocol can be grpc or http
+	// HTTPNetwork can be tcp, udp or unix
+	HTTPNetwork string
+	// HTTPAddr to listen on, hostname:port (0.0.0.0:9100 for all interfaces) or socket (/var/run/reva/sock)
+	HTTPAddr string
 	// Protocol can be grpc or http
 	Protocol string
-	// URL is used by the gateway and registries (eg http://localhost:9100 or https://cloud.example.com)
-	URL string
+	// Endpoint is used by the gateway and registries (eg localhost:9100 or cloud.example.com)
```

Comment:
this is the best I could come um with ... an endpoint is a combination of a tcp host and a port ... try googling for it. the only two URLs are the public facing endpoints, which is why the are now called `PublicURL`